client/src/components/About/About.jsx

```diff
@@ -1,29 +1,62 @@
 import React from 'react';
```

Comment:
changes to this page seem to have messed with the spacing in paragraphs
https://puu.sh/DgKNg/6f08f12362.png